tests/js/server/aql/aql-queries-optimizer-sort.js

```diff
@@ -4,8 +4,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief tests for query language, sort optimizations
 ///
```

Comment:
why is this one removed from cluster to non-cluster? or is this just a github diff artifact?
 196  ...aql/aql-queries-optimizer-sort-cluster.js → .../server/aql/aql-queries-optimizer-sort.js

Reply:
We had several tests that were split into cluster and non-cluster.
That was done years ago when single server and cluster had vastly different feature sets.
For most of the tests this split wasn't justified anymore, and we can actually run the same test in both cluster and non-cluster mode.

Reply:
OK.